app/config/sources.inc.php

```diff
@@ -263,6 +263,10 @@
         'supported_locales' => $mozillaorg,
     ],
     'firefox/products/developer.lang' => [
+        'deadline'          => '2017-10-06',
```

Comment:
Why are you setting a deadline for this page? You should remove this and mark it as obsolete instead.